docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -57,6 +57,10 @@ testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
   * [--color](#--color)
   * [--no-color](#--no-color)
 
+When you execute the `testcafe` command, TestCafe first reads settings from the `.testcaferc.json` [configuration file](configuration-file.md) if this file exists. Then it applies settings provided in the command line. Command line settings override values from the configuration file in case they differ. TestCafe prints information about every overridden property in the console.
```

Comment:
You want to add marks to the command syntax line (the very first line)?
Well, I wouldn't do that. It would indicate that you can go completely without these arguments being specified (before we even talk about the configuration file).

I don't want to lead readers into thinking that these arguments are actually optional. They are not, we are just providing a new way to specify them.

That's why I added a note about them being optional after the paragraph about the configuration file.